### ISSUE TITLE
train vector copy

### ISSUE THREAD

**Issue by linamede** [Mostafa-Samir/DNC-tensorflow#6]
Trying to train a model  that copies a 10x1 vector, the loss converges, however the read weights seem to be fade. Why is this happening? (I changed  memory_words_num to 20, and   memory_word_size to 20)

![train-series_10x1_tar txt_min_avg_loss_0 0045](https://cloud.githubusercontent.com/assets/15981686/21813882/465c38d2-d760-11e6-9f96-6730863264a3.png)

![train-series_10x1_tar-50000](https://cloud.githubusercontent.com/assets/15981686/21813887/4a386714-d760-11e6-8d3d-94eb4d50433d.png)

**Comment by Mostafa-Samir** [comment]
Hmmm ... interesting!

It could be possible because the input is small and needs only one memory location, the neural network was able to learn to copy it directly without needing to query the data it wrote in memory. you can check that by running the same trained model with only 1 memory location and see if it still outputs correctly.

I'll keep this issue opened for sometime to understand what's happening!

**Comment by linamede** [comment]
Using a memory of size one resulted in an error during the visualization...
Finally to train a vector, I used the train.py instead of train-series.py, which trains with random length. This worked and resulted in something like this. Thank you!

![train-40000_length_1](https://cloud.githubusercontent.com/assets/15981686/22016453/081883d8-dcaf-11e6-9dc9-1d5fa09ddb29.png)

**Comment by Mostafa-Samir** [comment]
I believe that error in the visualization can be avoided by tweaking the `visualize_op` method a little. Anyway, I'm glad you got your results eventually!
